app/scripts/models/reliers/oauth.js

```diff
@@ -16,9 +16,49 @@ define(function (require, exports, module) {
   var RelierKeys = require('lib/relier-keys');
   var Url = require('lib/url');
   var Validate = require('lib/validate');
+  var Vat = require('lib/vat');
 
   var RELIER_FIELDS_IN_RESUME_TOKEN = ['state', 'verificationRedirect'];
 
+  /*eslint-disable camelcase*/
+  var CLIENT_INFO_SCHEMA = {
+    id: Vat.hex().required().as('clientId'),
```

Comment:
I prefer `renameTo` to the simpler `rename` because the direction is the bit that confused me. Anything with the `To` suffix sounds good to me.